app/controllers/additional_submission_informations_controller.rb

```diff
@@ -0,0 +1,48 @@
+# voluntary submission information controller for creation only
+class AdditionalSubmissionInformationsController < ApplicationController
+  def new
+    @info = AdditionalSubmissionInformation.new(ori: find_ori)
+  end
+
+  def create
+    @info = AdditionalSubmissionInformation.new(info_params)
```

Comment:
including Serializable module took care of it, did not need to add the constants module.